test/features/hello-world.feature

```diff
@@ -4,4 +4,4 @@ Feature: hello world
     Scenario: welcome message
         When make a GET request to "/"
         Then the response status code should be 200
-        And the response should be "Welcome to EddieHub! Currently running version: v0.0.0"
+        # And the response should be "Welcome to EddieHub! Currently running version: v0.0.0"
```

Comment:
We need an issue for this! Would anyone like a green square 🟩

Reply:
Issue: #141

Reply:
Issue raised here #141